### PR TITLE
Fix FAQ icons

### PR DESCRIPTION
**Pull request by ashwindevx**
I wanted to commit only `Fix faq icons` but there is `updated files` commit too and I don't how to delete/remove it.

So I only want `Fix faq icons` commit to be reviewed.

Thanks!

### CODE REVIEW

**Review by Rucha1499 — COMMENTED** [review]
Congratulations on your first PR in RDS! 🎉 

Can you add a short video/ gif to the description of the PR?  It will be easy to visualize.

**Comment by ashwindevx** [comment]
@Rucha1499 I made these icons change when clicking.
ezgif com-gif-maker

File: js/faq.js [hunk 1]
```diff
@@ -1,7 +1,15 @@
 const faqTitles = document.querySelectorAll(".faq__title");
+const plusIcon = document.querySelectorAll('.plus-img');
+const minusIcon = document.querySelectorAll('.minus-img');
 
-faqTitles.forEach((faqTitle) => {
-  faqTitle.addEventListener("click", (event) => {
+faqTitles.forEach(function (eachBtn) {
+  eachBtn.addEventListener('click', function(e) {
```
Comment (tejasberi) [inline js/faq.js:6]: do we need a removeEventListener at some point?

**Review by swarajpure — CHANGES_REQUESTED** [review]
Can you please revert the changes in the files which are not supposed to be a part of this PR?

I can see that the commit `Merge branch 'develop' into develop` is removing a couple of files which are required for this project.

As of now, you can either revert that particular commit or you could directly copy paste the whole file which is not related to your task, that way there will be less number of files changed since the develop branch and your branch will be having the same version of files which are not related to your task.

Thank you! 🎉

**Comment by ashwindevx** [comment]
image
I'm getting this error. I don't know how to fix it.

**Comment by swarajpure** [comment]
> 
> 
> image
> I'm getting this error. I don't know how to fix it.

I don't know what app you're using.
You could try the other solution which I have suggested in the review 👍🏻 

> you could directly copy paste the whole file which is not related to your task, that way there will be less number of files changed since the develop branch and your branch will be having the same version of files which are not related to your task.

Commit and push that, let's see if that works.

**Comment by swarajpure** [comment]
Closing as the author did not respond in 2 days

**Pull request closed without merging**